### ISSUE TITLE
Rename or fix isJobComplete

### ISSUE THREAD

**Issue by sammefford** [marklogic/java-client-api#622]
The current implementation for JobReport.isJobComplete calls batcher.isStopped().  Those have different meanings.  Let's either rename JobReport.isJobComplete to JobReport.isJobStopped() or add isJobComplete to each batcher.

**Comment by georgeajit** [comment]
I am not sure what needs to be tested. The method is still available. One of the QA functional test still have the ```isJobStopped()``` method. And I don't see ```isJobComplete``` on QueryBatcher.

**Comment by sammefford** [comment]
Hmm, commit 113b83e seems to totally miss deleting the method.   Let me try again.

**Comment by srinathgit** [comment]
Verified that isJobComplete has been removed.
